web/src/components/pair/SimilarityDisplay.vue

```diff
@@ -40,6 +40,7 @@ interface Props {
   progress?: boolean;
   dense?: boolean;
   dimBelowCutoff?: boolean;
+  showGreen?: boolean;

```

Comment:
This value is not used, and the locations where we'll use this `showGreen` will have the same functionality as `dimBelowCutoff`, so these props may be merged?

Reply:
This prop is unused, so it can be removed